src/main/java/com/github/cchacin/cucumber/steps/RedisSteps.java

```diff
@@ -0,0 +1,14 @@
+package com.github.cchacin.cucumber.steps;
+
+import cucumber.api.java.en.Given;
+import redis.clients.jedis.Jedis;
+
+public class RedisSteps {
+    private final Jedis jedis = new Jedis("localhost");
+
+    @Given("^I have cleaned db (\\d+)$")
```

Comment:
lol, you are so right, i should add redis.

So you think without db specified, it means we want to flush all the dbs?